cmd/antrea-agent/agent.go

```diff
@@ -864,6 +865,14 @@ func run(o *Options) error {
 			return fmt.Errorf("failed to connect uplink to OVS bridge: %w", err)
 		}
 	}
+	// secondaryNetworkController Initialize must be run after FlowRestoreComplete.
```

Comment:
Add "for the case that Node IPs are moved to the secondary OVS bridge"?

Reply:
added.